src/Command/PluginLoadCommand.php

```diff
@@ -110,7 +110,7 @@ protected function modifyApplication(string $app, string $plugin): void
         }
 
         // Check if plugin is already loaded
-        $regex = '/->addPlugin\(\'' . $plugin . '\'/mu';
+        $regex = '#->addPlugin\(\'' . $plugin . '\'#mu';
```

Comment:
The alternative is quoting $plugin

Reply:
Yes but this saves a method call and plugin names are not going to have any special chars besides slash :)